compose.prod.yaml

```
@@ -0,0 +1,23 @@
+services:
+  lyra:
+    container_name: lyra
+    image: ghcr.io/zetkin/lyra:0.4.1
+    environment:
+      - GIT_USER_EMAIL=lyra@zetkin.org
+      - GIT_USER_NAME="Lyra Translator Bot"
+    volumes:
+      # note: adjust the path to the ssh key you want to use to access the git repositories
+      - ~/.ssh/id_rsa:/home/nodeuser/.ssh/id_rsa:ro
+      # note: adjust the path to the lyra-store-file you want to use
+      - ~/lyra-store.json:/app/webapp/store.json
+      - ./config:/app/config
+  nginx:
+    image: nginx:stable
+    container_name: nginx-reverse-proxy
+    restart: unless-stopped
+    ports:
+      - "80:80"
+      - "443:443"
```

For nginx to be able to connect (and proxy) to lyra, there needs to be a `links` property here, I believe

```suggestion
  nginx:
    image: nginx:stable
    container_name: nginx-reverse-proxy
    restart: unless-stopped
    links:
      - lyra:lyra
    ports:
      - "80:80"
      - "443:443"
```

At least that was required for it to work when I was trying it.

---

Links are not required to enable services to communicate. When no specific network configuration is set, any service is able to reach any other service at that service’s name on the default network.

https://docs.docker.com/reference/compose-file/services/#links

But on the other hand, it doesn't hurt either to add a links property.